xml/System.CodeDom.Compiler/CodeGenerator.xml

```diff
@@ -2715,7 +2715,7 @@
       </Parameters>
       <Docs>
         <param name="e">The statement to generate code for.</param>
-        <summary>Generates code for the specified <see langword="try...catch...finally" /> statement.</summary>
+        <summary>Generates code for the specified "try-catch-finally" statement.</summary>
```

Comment:
@NextTurn What is the reasoning behind these changes?

Reply:
Ellipses are not keywords.

Reply:
How about inline code then to prevent them from being localized?

```suggestion
        <summary>Generates code for the specified <c>try-catch-finally</c> statement.</summary>
```

Reply:
Makes sense, thanks!